nixos/modules/tasks/network-interfaces-scripted.nix

```
@@ -206,14 +206,14 @@ let
                     if out=$(ip addr replace "${cidr}" dev "${i.name}" 2>&1); then
                       echo "done"
                     else
-                      echo "'ip addr replace "${cidr}" dev "${i.name}"' failed: $out"
+                      echo "'ip addr replace \"${cidr}\" dev \"${i.name}\"' failed: $out"
```

I did this:
```suggestion
                      echo "'ip addr replace ${cidr} dev ${i.name}' failed: $out"
```
but I think your variant is more correct when I think about it